components/QuestionCard.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+
+export type QuestionCardProps = {
+  number: number;
+  question: string;
+  answers: { answer: string; isTrue: boolean; key: number }[];
+};
+
+export const QuestionCard = ({
+  number,
+  question,
+  answers,
+}: QuestionCardProps) => {
+  const renderedAnswers = answers.map(({ answer, isTrue, key }) => {
+    return <li key={key}>{answer}</li>;
```

Comment:
Since you were asking about the key property. The only requirement for a key is that it is unique per entry in the array. Here you could just use the answer property a react key.

Reply:
```suggestion
    return <li key={answer}>{answer}</li>;
```